File: 프로그래머스/1/12948. 핸드폰 번호 가리기/핸드폰 번호 가리기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string phone_number) {
    string answer = "";

    int pn_length = phone_number.length();
    for(int i=0; i<pn_length-4; i++) answer += "*";
    for(int i=0; i<4; i++) answer += phone_number[pn_length-4+i];
    
    return answer;
}

File: 프로그래머스/1/12948. 핸드폰 번호 가리기/핸드폰 번호 가리기.js
function solution(phone_number) {
    var answer = '';
    let count = 0;
    
    for(let i=phone_number.length -1 ; i>=0; i--) {
        if (count >= 4) answer = '*' + answer;
        else answer = phone_number[i] + answer;
        count += 1;
    }
    
    return answer;
}